### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by eabase** [pypa/setuptools#3680]
### setuptools version

65.5.1

### Python version

Python 3.10.0

### OS

Win-10

### Additional environment information

pip 22.3.1

### Description

I'm trying to supply an example script as part of the package. 
The project tree is: 
```bash
# tree -L 3
.
├── docs
├── examples
│   └── fbs_test.py
├── src
│   ├── black_scholes
│   │   └── __init__.py
│   └── lib
│       ├── Makefile
│       └── fbs.c
├── .gitignore
├── LICENSE.md
├── README.md
├── clean.sh
├── pyproject.toml
└── setup.py
```

The `pyproject.toml` looks like this:
```toml
# ...
[project.scripts]
"fbs_test.py" = "examples:fbs_test.py"

[tool.setuptools]
package-dir = {"" = "src"}

[tool.setuptools.packages.find]
where = ["src"]
```

### This have 2 problems: 

1. the included file is named wrong: `fbs_test.exe`, instead of `fbs_test.py`
2. Running the file is looking for a module names 'examples' which is totally worng.  
`ModuleNotFoundError: No module named 'examples'`

### Expected behavior

To be able to run the installed `fbs_test.py` with:

```powershell
$ fbs_test.py

Testing our Black-Scholes() with a 60 day expiration:
black-scholes(147.3, 150.0, 0.001, 0.1643835616438356, 0.45) = 9.518562265392418
```

### How to Reproduce

`pip install . --no-cache-dir`

### Actual behavior

```powershell
# fbs_test.exe

Traceback (most recent call last):
  File "C:\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Python310\Scripts\fbs_test.exe\__main__.py", line 4, in <module>
ModuleNotFoundError: No module named 'examples'
```

**Comment by abravalheri** [comment]
Hi @eabase, regarding problem 2 please note that `pyproject.toml`'s `[project.scripts]` should be used to specify [entry-points](https://packaging.python.org/en/latest/specifications/entry-points/) and not script files.

This means that if you write `"fbs_test.py" = "examples:fbs_test.py"` what will happen is that the script wrapper (which is not generated by `setuptools`, but rather `pip` with help of `distlib`) will try to load a `py` function from a `fibs_test` module inside a `examples` package.

This does not seem to correspond to your configuration/project layout.


I recommend trying the following:

```bash
# tree -L 3
.
├── docs
├── src
│   ├── black_scholes
|   |   └── examples
|   |       └── fbs_test.py
|   |           └── __init__.py
│   └── lib
│       ├── Makefile
│       └── fbs.c
├── .gitignore
├── LICENSE.md
├── README.md
├── clean.sh
├── pyproject.toml
└── setup.py
```

```toml
# ...
[project.scripts]
"fbs_test" = "black_scholes.examples:fbs_test.main"
             # ^-- here I am supposing that a `main` function
             #     exists inside black_scholes/examples/fbs_test.py

# ...
```

Regarding problem 1, please note that `setuptools` is not being used to generate the executable wrapper in this scenario. Instead they are created by `pip` with help of `distlib`.

Also note that there is not much you can do about the `.exe` suffix in Windows. Executable files on Windows need an executable suffix (e.g. `.bat` or `.exe`). The good news is that you can omit the `.exe` part when typing the name of your command, and it should work.

**Comment by eabase** [comment]
Hi @abravalheri 

Thank you for prompt reply! 

I have indeed moved the script inclusion from the `*.toml` file into the old school `setyp.py`.  I had to do this, because the suggested `tree` does not look good, from a user experience perspective.  In this regard, I fail to see how setup.py is ever going to be replaced by the `*.toml` approach, if it doesn't allow for simple script/data file inclusion, using the whatever tree structure you have.
